server/service/schema/transactions.schema.js

```diff
@@ -0,0 +1,67 @@
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+
+const transaction = new Schema(
+  {
+    date: {
+      // TODO co przychodzi z kompnentu?
+      //type: String,
+    },
+    year: {
+      type: Number,
+    },
+    month: {
+      type: String,
+      enum: [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December',
+      ],
+    },
+    type: {
```

Comment:
Dodałbym jeszcze pole, że typ jest wymaganym elementem schematu: require:[true, 'Type of transaction is required'] :)